src/eventHandlers/ResourceBookingEventHandler.js

```diff
@@ -84,20 +84,22 @@ async function processUpdate (payload) {
   if (payload.status !== 'assigned') {
     logger.info({
       component: 'ResourceBookingEventHandler',
-      context: 'selectJobCandidate',
+      context: 'processUpdate',
       message: `not interested resource booking - status: ${payload.status}`
     })
     return
   }
   const resourceBooking = await models.ResourceBooking.findById(payload.id)
-  const job = await models.Job.findOne({
+  const jobs = await models.Job.findAll({
```

Comment:
@imcaizheng ResourceBooking is associated with one Job using `jobId`, see https://github.com/topcoder-platform/taas-apis/blob/dev/src/models/ResourceBooking.js#L54. So we don't need to search for all jobs in the project.

Reply:
I guess we could make` jobId` a required value. See https://github.com/topcoder-platform/taas-apis/blob/dev/src/services/ResourceBookingService.js#L123, I are not sure it is by intention or just a careless mistake. Otherwise, since `jobId` is optional, we cannot relay on that.